src/www/diag_dns.php

```diff
@@ -77,7 +77,7 @@
 			$newalias['name'] = $aliasname;
 			$newalias['type'] = "network";
 			$newalias['address'] = $addresses;
-			$newalias['descr'] = "Created from Diagnostics-> DNS Lookup";
+			$newalias['descr'] = gettext("Created from Diagnostics-> DNS Lookup");
```

Comment:
Please always use single quotes for gettext strings (like above) -- they cannot be dynamic anyway and it's a little faster this way.